nncf/parameters.py

```diff
@@ -89,6 +89,19 @@ class CompressWeightsMode(Enum):
     INT8 = "int8"  # Deprecated mode
 
 
+class MixedPrecisionMode(Enum):
+    """
+    Defines a mode for selecting quantization precision.
+    :param : TODO:
+    """
+
+    INT8_ERROR = "int8_error"
```

Comment:
`INT8_ERROR` -> `WEIGHT_QUANTIZATION_ERROR`
`HAWQ_IN` -> `HESSIAN_INPUT_ACTIVATION` or `HAWQ_INPUT_ACTIVATION`
`MEAN_VAR` -> `MEAN_ACTIVATION_VARIANCE`
`MAX_VAR` -> `MAX_ACTIVATION_VARIANCE`
`MEAN_MAX` -> `MEAN_ACTIVATION_MAGNITUDE`

Reply:
renamed, selected `HESSIAN_INPUT_ACTIVATION` for `HAWQ_IN`